src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas2/OffenderServiceTest.kt

```diff
@@ -273,17 +296,29 @@ class OffenderServiceTest {
 
       val inmateDetail = InmateDetailFactory()
         .withOffenderNo(nomsNumber)
+        .withAssignedLivingUnit(
```

Comment:
Just curious what has changed that we need to add an AssignedLivingUnit?

Reply:
we are checking `isOffenderSamePrisonAsUser`  which compares `assignedLivingUnit.agencyId` of the InmateDetails to the user's `activeCaseloadId`. so we need to set the inmate details up with the same prison as the user  for the happy path